libs/features/mas/test/merch-card-collection.test.html.js

```diff
@@ -203,7 +203,7 @@ runTests(async () => {
             document.location.hash = 'single_app=illustrator';
             render();
             await delay(100);
-            expect(visibleCards(1).name).to.equal('illustrator');
+            expect(visibleCards(1).getAttribute('card-name')).to.equal('illustrator');
```

Comment:
`card.getAttribute` should not be needed. If `card.name` doesn't work that means something needs to be fixed.

Reply:
I know card.cardName should work for most modern browsers,  but for some reason, they are suddenly failed yesterday. I tested today again and worked now. So weird.

Reply:
it might be an issue with the loading of merch-card custom element.